oolab/src/main/java/agh/ics/oop/model/GrassField.java

```diff
@@ -0,0 +1,38 @@
+package agh.ics.oop.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class GrassField extends AbstractWorldMap {
```

Comment:
brakuje `objectAt()`, które uwzględniłoby to czy jest na polu zwierzak czy roślina.